website/content/en/docs/advanced-topics/heterogeneous.md

```diff
@@ -0,0 +1,400 @@
+---
+title: Heterogeneous Cluster Support
+linkTitle: Heterogeneous Cluster Support
+weight: 80
+---
+## Overview
+A heterogeneous cluster is a cluster in which the nodes differ from each other in a way that makes their containers incompatible with each other. The most common incompatibility that applies is CPU architecture. The rest of this document assumes that the user is developing an application that needs to be able to deploy cleanly in a cluster with a mix of node CPU architectures. Because an operator deployed to a heterogeneous cluster may not support all of the architectures in the cluster, it is important to set up safeguards that prevent scheduling pods on nodes of incompatible architectures. These scheduling implications affect both the operator as well as its operands.
+
+### Determining the Supported Architectures for an Operator/Operand Image
+For a given container image, you can check which platforms are supported by listing them by inspecting the manifest.
+```bash
+docker manifest inspect <image>
+```
+If the above command fails, it may be because the referenced name points to a single image rather than a manifest list. In this case, you can find the architecture by inspecting the image.
+```bash
+docker inspect <image>
+```
+
+Here's an example of the architectures distributed for the Alpine Linux container image.
+```bash
+$ docker manifest inspect alpine | grep architecture
+            "architecture": "amd64",
+            "architecture": "arm",
+            "architecture": "arm",
+            "architecture": "arm64",
+            "architecture": "386",
+            "architecture": "ppc64le",
+            "architecture": "s390x",
+```
+
+#### Shipping a Multi-Architecture Operator
+If once you've determined which subset of architecture nodes you want to distribute your operator for, there are tools available to help you distribute your operator.
+
+OperatorSDK provides a validator to help check bundle configuration.
+```bash
+operator-sdk bundle validate ./bundle --select-optional name=multiarch
+```
+
+If you're planning to distribute the operator via OLM, you can find more information in the [OLM guide for supporting multiarch](https://olm.operatorframework.io/docs/advanced-tasks/ship-operator-supporting-multiarch/).
+
+### Setting the Supported Architectures Operator & Operand Workloads
```

Comment:
```suggestion
### Setting the Supported Architectures on the Operator & Operand Workloads
```
This heading feels like it needs something. Either "on the Operator & ..." or "for the Operator & ..."